contracts/examples/VCPayment-v2.sol

```diff
@@ -0,0 +1,278 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.8.20;
+import {Ownable2StepUpgradeable} from "@openzeppelin/contracts-upgradeable/access/Ownable2StepUpgradeable.sol";
+
+contract VCPaymentV2 is Ownable2StepUpgradeable {
+    /**
+     * @dev Version of contract
+     */
+    string public constant VERSION = '2.0.0';
+
+    /**
+     * @dev Payment Data structure
+     */
+    struct PaymentData {
+        uint256 issuerId;
+        uint256 schemaHash;
+        uint256 valueToPay;
+        uint256 ownerPartPercent;
+        address withdrawAddress;
+        // for reporting
+        uint256 totalValue;
+    }
+
+    /**
+     * @dev Main storage structure for the contract
+     */
+    struct Data {
```

Comment:
1. The NatSpec tag is better to put here. That's according to the standard and can support some dev tooling maybe.
`/// @custom:storage-location erc7201:iden3.storage.VCPayment`

2. Better to rename it to `VCPaymentStorage` for naming unification